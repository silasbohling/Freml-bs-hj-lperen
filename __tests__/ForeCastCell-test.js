import React from 'react'
import ForeCastCell from '@components/ForeCastCell'

import Renderer from 'react-test-renderer'

describe( 'ForeCastCell', () => {

  it('render correctly', () => {
    const forecast = {
      time: 389485,
      temp: 3.5,
      setting: 6,
      chillFactor: 12.3,
      chillFactorSetting: 8
    }

    const component = <ForeCastCell forecast={ forecast } />
    const three = Renderer.create(component).toJSON()

    expect(three).toMatchSnapshot()
  })
})
