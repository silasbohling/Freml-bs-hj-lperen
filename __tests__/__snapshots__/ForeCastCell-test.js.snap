// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ForeCastCell render correctly 1`] = `
<View
  style={
    Object {
      "borderBottomColor": "rgba(0,0,0,0.3)",
      "borderBottomWidth": 1,
      "paddingHorizontal": 2,
      "paddingVertical": 7,
      "width": "100%",
    }
  }
>
  <View
    style={
      Object {
        "flexDirection": "row",
        "justifyContent": "space-between",
        "width": "100%",
      }
    }
  >
    <Text
      style={
        Array [
          Object {
            "fontSize": 16,
          },
        ]
      }
    >
      5/1 12:11
    </Text>
    <View
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      <Text
        style={
          Array [
            Object {
              "fontSize": 16,
            },
            Object {
              "fontWeight": "600",
              "marginRight": 5,
            },
          ]
        }
      >
        3.5 C°
      </Text>
      <Text
        style={
          Array [
            Object {
              "fontSize": 16,
            },
          ]
        }
      >
        (
        6
        )
      </Text>
    </View>
  </View>
  <View
    style={
      Object {
        "flexDirection": "row",
        "justifyContent": "space-between",
        "width": "100%",
      }
    }
  >
    <Text
      style={
        Array [
          Object {
            "fontSize": 16,
          },
        ]
      }
    />
    <View
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      <Text
        style={
          Array [
            Object {
              "fontSize": 16,
            },
          ]
        }
      >
        12.3 C°
      </Text>
      <Text
        style={
          Array [
            Object {
              "fontSize": 16,
            },
          ]
        }
      >
        (
        8
        )
      </Text>
    </View>
  </View>
</View>
`;
